device-types/Cisco/N9K-C9372TX-E.yaml

```diff
@@ -0,0 +1,152 @@
+---
+manufacturer: Cisco
+model: Nexus 9372TX-E
+part_number: N9K-C9372TX-E
+slug: n9k-C9372tx-e
+u_height: 1
+is_full_depth: true
+########################################################################################################
+### airflow: This is configurable based on what Fan and Power Supply modules you have in your device ###
+########################################################################################################
+weight:
+  - value: 22.6
+    unit: lb
+comments: '[Cisco Nexus 9300 Platform Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-9000-series-switches/datasheet-c78-736967.html)'
+console-ports:
+  - name: console
+    type: rj-45
+module-bays:
+  - name: Slot 1
+    label: 'N9K-C9372TX-E'
+    position: '1'
+    description: '48x1/10G-T 6x40G Ethernet Module'
+  - name: Power Supply 1
+    position: '2'
+    description: 'Nexus9000 C9372TX chassis Power Supply'
+  - name: Power Supply 2
+    position: '3'
+    description: 'Nexus9000 C9372TX chassis Power Supply'
+  - name: Fan 1
+    position: '4'
+    description: 'Nexus9000 C9372TX chassis Fan Module'
+  - name: Fan 2
+    position: '5'
+    description: 'Nexus9000 C9372TX chassis Fan Module'
+  - name: Fan 3
+    position: '6'
+    description: 'Nexus9000 C9372TX chassis Fan Module'
+  - name: Fan 4
+    position: '7'
+    description: 'Nexus9000 C9372TX chassis Fan Module'
+interfaces:
+  - name: Ethernet{module}/1
```

Comment:
Any reason these have `{module}` in them? This is a device definition file, not a module definition file. If these are not modular ports then they should have the actual value, i.e. `Ethernet1/1`. Also, if you run a `show config` or other applicable command, this should read exactly as the devices config does. On a 9200L for instance, even though you can configure a port by using `int gi1/0/1` it should be typed in this definition file as `GigabitEthernet1/0/1`. I have never used one of these devices so I dont know, just want to mention that in case it applies here.

Reply:
I have removed {module} and replaced it with the numerical value as instructed.